### ISSUE TITLE
bug: Moving item from Chocobo Saddlebag to other containers does not register item

### ISSUE THREAD

**Issue by anthonypants** [ffxiv-teamcraft/ffxiv-teamcraft#1712]
## I'm submitting a...
* (X) Bug report *- please add **bug:** before the title*

## Environment
Windows 10 20H2 (Build 19042.610)
### Browser
ffxivteamcraft.exe version 7.5.4

## Current behavior
Inventory is not updated when moving items between Chocobo Saddlebag and other containers (e.g. Inventory, Armoury Chest). Additionally, this appears to break all future inventory updates, no matter what container is affected (e.g. FC Chest, Retainers).

## Expected behavior
Inventory is updated in Teamcraft when moving items between Chocobo Saddlebag and other containers; moving items between Chocobo Saddlebag and other containers does not prevent updates the status of all other containers.

## Steps to reproduce
1. Log in to Final Fantasy XIV.
2. Open FFXIV Teamcraft.
3. Open up the Dev Tools window in FFXIV Teamcraft.
4. Move an item from your Chocobo Saddlebag into your Inventory.
5. Move the same item back from your Inventory into your Chocobo Saddlebag.
6. The Dev Tools pane should show a "Tried to move an item that isn't registered in inventory" warning.
7. Moving this item at all will result in additional warnings (e.g. from one cell in your Chocobo Saddlebag to another cell in your Chocobo Saddlebag).
8. In Teamcraft, searching for the item will report that it is in the Chocobo Saddlebag, regardless of where it is moved.
9. Moving items from any other container (e.g. from your Armoury Chest to your Inventory, to/from your FC Chest or retainers) will not update the inventory list in Teamcraft.
10. Performing a reset on your inventory in Teamcraft _may_ resolve this issue. If not, uninstall FFXIV Teamcraft, remove the `%appdata%\ffxiv-teamcraft` and `%localappdata%\ffxiv-teamcraft` folders, **reboot**, and reinstall FFXIV Teamcraft.

**Comment by Supamiu** [comment]
This seems to be due to the `InventoryModifyHandler` flag being wrong, TC is currently using `0x34A` while it should be `0x13C`

**Comment by anthonypants** [comment]
I'm getting the `Tried to move an item that isn't registered in inventory` message in the dev tools pane when moving an item from my Inventory into FC Chest, but **only** when I'm adding to a stack that already exists in the FC Chest.

For example, I've got some Coerthan Tea Leaves that I just finished gathering, and I want to place them in the FC Chest, where there is already a stack of Coerthan Tea Leaves, so I drop them in and Teamcraft gives me an `Tried to move an item that isn't registered in inventory` error. I can also remove a partial stack from the FC Chest and replace it to reproduce this warning. Additionally, I can reproduce this warning on all FC Chest containers **except** FC Chest #1.

I thought this might have been related to the earlier error, so I reset my inventory, but that didn't affect this bug. Also, unlike the above bug, this does **not** break synchronization/updating of the Teamcraft inventory. And, of course, I updated to 7.5.5 this morning.
